msg/DifferentialDriveSetpoint.msg

```diff
@@ -0,0 +1,5 @@
+uint64 timestamp	# time since system start (microseconds)
```

Comment:
Don't forget to add it to `logged_topics.cpp`, otherwise this new topic would not get logged.

Reply:
Will do, is there a certain frequency rate [hz] convention that I should follow?

Reply:
Alright, perfect!